recipe/meta.yaml

```diff
@@ -1,38 +1,38 @@
 {% set name = "deepchem" %}
-{% set version = "2.3.0" %}
+{% set version = "2.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/deepchem/deepchem/archive/{{ version }}.tar.gz
-  sha256: eec0d4dfa23e234d26d171ef20c62a699052a1bc230bbace341047e0c2673030
+  sha256: 25ebc59a53c8dcd841dbebab00bbef85efe8c29fa33e49c36b751cbd2f76110c
 
 build:
   noarch: python
-  number: 2
-  script: "PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  number: 0
+  script: PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
 
 requirements:
   host:
-    - python >=3.5
+    - python >=3.5,<3.8
     - pip
     - setuptools
     - pbr
 
   run:
-    - python >=3.5
+    - python >=3.5,<3.8
+    - pip
     - joblib
     - numpy
     - pandas
-    - pillow # pillow will remove from v2.4.0
-    - scikit-learn <=0.22.1
+    - scikit-learn
     - scipy
 
 test:
   requires:
```

Comment:
if the tests are being removed, this `requires` section can also be removed / commented out, or at least tensorflow